src/types/__tests__/profile.test.ts

```diff
@@ -20,10 +20,12 @@ import { Version } from "../../../generated/backend/Version";
 
 import { ExtendedProfile as ExtendedProfileApi } from "../../../generated/io-api/ExtendedProfile";
 
+import { AcceptedTosVersion } from "../../../generated/backend/AcceptedTosVersion";
 import { toAuthenticatedProfile, toInitializedProfile } from "../profile";
 import { SessionToken, WalletToken } from "../token";
 import { User } from "../user";
 
+const aTosVersion = 1 as AcceptedTosVersion;
```

Comment:
you should create a test to check what happens when the functions API returns `accepted_tos_version === undefined`

Reply:
Add test to manage undefined ToS version response from functions API